src/main/java/io/appform/functionmetrics/FunctionTimerAspect.java

```diff
@@ -43,8 +52,35 @@ public Object around(ProceedingJoinPoint joinPoint) throws Throwable {
         final String methodName = Strings.isNullOrEmpty(monitoredFunction.method())
                                     ? callSignature.getName()
                                     : monitoredFunction.method();
-        log.trace("Called for class: {} method: {}", className, methodName);
-        final FunctionInvocation invocation = new FunctionInvocation(className, methodName);
+        final String[] parameterNames = methodSignature.getParameterNames();
+        String parameterString = "";
+        List<String> paramValues = Streams.zip(Arrays.stream(methodSignature.getMethod().getParameters()), Arrays.stream(joinPoint.getArgs()), Pair::new)
```

Comment:
This is super expensive. Check flag before dabbling with the dark arts.

Reply:
My bad, added it now.